src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -530,28 +548,36 @@ private void writePortsToFile(final Config config, final int nodePort, final int
     };
   }
 
-  private KeyValueStore createStorage(final String storage, final Path storagePath) {
-    String db = "routerdb";
+  private KeyValueStore<Bytes, Bytes> createStorage(final String storage, final Path storagePath, String dbName) {
```

Comment:
I know that this isn't technically part of this PR. But it would be great if we could move this logic of creating the storage to its own class. Feel free to ignore this comment if you think it will take up too much of your time :)

Reply:
Maybe another PR? Got a bunch of things going in there already. Happy to as a way to clean things up indeed.

Reply:
Sure thing! It doesn't need to be part of this.